### ISSUE TITLE
Setting Up Multiple accounts

### ISSUE THREAD

**Issue by jackfivefive** [mifi/SimpleInstaBot#217]
Hi, could you describe more how to run multiple accounts in the same time.
I am trying to do it but it is not clear enough for me.

**Comment by mifi** [comment]
hi. see https://github.com/mifi/SimpleInstaBot#multiple-accounts
see also previous issues

**Comment by nasw1h** [comment]
Yeah, I'm too trying to run two accounts same time, but not clear what to do.
![Screenshot 2023-04-02 at 07 40 18](https://user-images.githubusercontent.com/51323070/229327274-e914da58-eb5c-4482-9311-b058b3f0f500.png)
